drivers/adc/ad7193/ad7193.c

```diff
@@ -319,6 +334,163 @@ void ad7193_calibrate(struct ad7193_dev *dev,
 	AD7193_CS_HIGH;
 }
 
+/***************************************************************************//**
+ * @brief Configures the input mode of the ADC
+ *
+ * @param dev   - The device structure.
+ * @param mode  - Mode selection.
+ *				  Example: 0 - Differential Analog Inputs
+ *						   1 - Pseudo Differential Analog Inputs
+ *
+ * @return none.
+*******************************************************************************/
+void ad7193_config_input_mode(struct ad7193_dev *dev, uint8_t mode)
+{
+	uint32_t old_reg_value = 0x0;
+	uint32_t new_reg_value = 0x0;
+
+	old_reg_value  = ad7193_get_register_value(dev,
+			 AD7193_REG_CONF,
+			 3,
+			 1);
+
+	old_reg_value &= ~(AD7193_CONF_PSEUDO);
```

Comment:
Why not adding a generic `ad7193_update_register value` similar to:
https://github.com/analogdevicesinc/no-OS/blob/a2f59796c297b03cf5f36a59d4d4d441b95b5230/drivers/adc/ad463x/ad463x.c#L111-L127

Most of these functions that your are adding seem to update specific bits of a register.

Reply:
Will implement this.